tests/unit/core/reducers/test_languageTools.js

```diff
@@ -19,28 +19,31 @@ describe(__filename, () => {
   });
 
   it('stores language tools', () => {
-    const language = createFakeLanguageTool();
+    const lang = 'fr';
+    const language = createFakeLanguageTool({ target_locale: lang });
+    const stateWithLang = reducer(undefined, setLang(lang));
```

Comment:
We shouldn't use the same `lang` and `target_locale` I think. We need a different `target_locale` than the current locale in order to reproduce the bug.